wave_containers/include/wave/containers/impl/landmark_measurement_container.hpp

```diff
@@ -19,7 +19,11 @@ using boost::multi_index::tag;
  * holding measurements of type T. See `wave::internal::measurement_container`.
  */
 template <typename T>
-struct landmark_container {
+struct container_traits<::wave::LandmarkMeasurementContainer<T>> {
+    // Define to give MeasurementContainerBase access to its derived type
+    using MeasurementType = T;
```

Comment:
Ah, right...because Base is templated with either the landmark or regular measurement containers.

I was actually wondering about where MeasurementType was defined in the base class; that makes a bit more sense now. Thanks